tests/baselines/reference/genericSetterInClassType.errors.txt

```diff
@@ -0,0 +1,38 @@
+tests/cases/conformance/classes/members/classTypes/genericSetterInClassType.ts(3,13): error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.
+tests/cases/conformance/classes/members/classTypes/genericSetterInClassType.ts(6,13): error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.
+tests/cases/conformance/classes/members/classTypes/genericSetterInClassType.ts(15,13): error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.
+tests/cases/conformance/classes/members/classTypes/genericSetterInClassType.ts(18,13): error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.
+
+
+==== tests/cases/conformance/classes/members/classTypes/genericSetterInClassType.ts (4 errors) ====
+    module NonGeneric {
+        class C {
+            get y() {
+                ~
+!!! error TS1056: Accessors are only available when targeting ECMAScript 5 and higher.

```

Comment:
Can you change the `target` on this test to avoid the noise from errors?

Reply:
Done.